docs/src/toml-files.md

```diff
@@ -0,0 +1,224 @@
+
+
+### The `uuid` field
+
+`uuid` is a string with a [universally unique identifier]
+(https://en.wikipedia.org/wiki/Universally_unique_identifier) for the package/project.
+Example:
+```toml
+uuid = "7876af07-990d-54b4-ab0e-23690620f79a"
+```
+The `uuid` field is mandatory for packages.
+
+
+### The `version` field
+
+`version` is a string with the version number for the package/project.
+Example:
+```toml
+version = "1.2.5"
+```
+Julia uses [Semantic Versioning](https://semver.org/) (SemVer) and the
+`version` field should follow SemVer. The basic rules are:
+* Before 1.0.0, anything goes, but when you make breaking changes
+  the minor version should be incremented.
+* After 1.0.0 only make breaking changes when incrementing the major version.
+* After 1.0.0 no new public API should be added without incrementing the
+  minor version. This includes, in particular, new types, functions, methods and
+  method overloads, from e.g. `Base` or other packages.
+
+
+### The `[deps]` section
+
+All dependencies of the package/project are listed in the `[deps]` section.
+Each dependency is listed as a name-uuid pair, for example:
+
+```toml
+[deps]
+Example = "7876af07-990d-54b4-ab0e-23690620f79a"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+```
+
+Typically it is not needed to manually add entries to the `[deps]` section,
+this is instead handled by Pkg operations such as `add`.
+
+
+### The `[compat]` section
+
+Compatibility constraints for the dependencies listed under `[deps]`
+can be listed in the `[compat]` section.
+Example:
+
+```toml
+[deps]
+Example = "7876af07-990d-54b4-ab0e-23690620f79a"
+
+[compat]
+Example = "1.2"
+```
+
+It is also possible to list constraints on `julia` itself, although `julia` is not
+listed as a dependency.
+
+```toml
+[compat]
+julia = "1.1"
+```
+
+The [Compatibility](@ref) section describes the different possible
+compatibility constraints in detail.
+
+
+## `Manifest.toml`
+
+The manifest file is an absolute record of the state of the packages in the environment.
+It includes exact information about (direct and indirect) dependencies of the project,
+and given a `Project.toml` + `Manifest.toml` pair it is possible to instantiate the
+exact same package environment, which is very useful for reproducibility,
+see [`Pkg.instantiate`](@ref).
+
+!!! note
+    The `Manifest.toml` file is generated by Pkg and in general this file
+    should never be modified manually.
+
+
+### `Manifest.toml` entries
+
+Each dependency has its own section in the manifest file, and its content varies depending
+on how the dependency was added, see the examples below.
+Every dependency section includes a combination of the following entries:
+
+* `uuid`: the [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier)
+  for the dependency, for example `uuid = "7876af07-990d-54b4-ab0e-23690620f79a"`.
+* `deps`: a vector listing the dependencies of the dependency, for example
+  `deps = ["Example", "JSON"]`.
```

Comment:
This is AFAIU only true if there is one dependency called `Example`. If there are multiple dependencies called `Example` a `name => uuid` Dict is instead printed.

Reply:
Yea, I can add a section about that maybe.